src/components/Game.js

```diff
@@ -64,6 +66,82 @@ function Game() {
     localStorage.setItem("contenttype", "Paragraph");
     localStorage.setItem("isfromresult", "learn");
     checkVoice(voiceText, Story[storyLine]);
+    generateResponseEvent(voiceText)
+  }
+
+
+  function generateResponseEvent(voiceText){
+   
+    let texttemp = voiceText.toLowerCase();
+    const studentTextArray = texttemp.split(" ");
+    let tempteacherText = localStorage.getItem("contentText")?.toLowerCase();
+
+    const teacherTextArray = tempteacherText?.split(" ");
+
+    let student_correct_words_result = [];
+    let student_incorrect_words_result = [];
+    let originalwords = teacherTextArray?.length;
+    let studentswords = studentTextArray?.length;
+    let wrong_words = 0;
+    let correct_words = 0;
+    let result_per_words = 0;
+    let result_per_words1 = 0;
+    let occuracy_percentage = 0;
+
+    let word_result_array = compareArrays(
+      teacherTextArray,
+      studentTextArray
+    );
+
+    for (let i = 0; i < studentTextArray?.length; i++) {
```

Comment:
looks like this logic is already implemented in checkVoice(). Please check it and reuse the code